backend/src/main/java/com/wootech/dropthecode/domain/review/Review.java

```diff
@@ -63,9 +61,21 @@ public Review(Member teacher, Member student, String title, String content, Stri
         this.progress = progress;
     }
 
+    private void validateAuthorityOfReview(Member teacher, Member student) {
```

Comment:
메소드 이름을 리뷰 생성에 대한 유효성 검증을 명시적으로 나타내면 더 좋을 것 같아요.

```suggestion
    private void validateAuthorityOfReviewCreation(Member teacher, Member student) {
```

지금은 리뷰 작업 모든 것에 대한 유효성 검사를 진행하는 것으로 보이네요.

Reply:
동의합니다 더 명시적이라서 의도파악하기가 쉽네요!!!